app/jobs/characterize_job.rb

```diff
@@ -2,9 +2,10 @@ class CharacterizeJob < ActiveJob::Base
   queue_as CurationConcerns.config.ingest_queue_name
 
   # @param [FileSet] file_set
-  # @param [String] filename a local path for the file to characterize. By using this, we don't have to pull a copy out of fedora.
+  # @param [String] filename a local path for the file to characterize so we don't have to pull a copy out of fedora.
   def perform(file_set, filename)
-    Hydra::Works::CharacterizationService.run(file_set, filename)
+    return unless file_set.characterization_proxy.content
```

Comment:
I followed the rabbit trail up to ActiveFedora::File and submitted an issue related to content? or some such thing. https://github.com/projecthydra/active_fedora/issues/1082

To resolve/answer my question, can you attach an `@todo Update characterization_proxy.content to not read content once https://github.com/projecthydra/active_fedora/issues/1082 is resolved`